BugsnagBundle.php

```diff
@@ -12,4 +14,16 @@ class BugsnagBundle extends Bundle
      * @return string
      */
     const VERSION = '1.0.0';
+
+    /**
+     * {@inheritdoc}
```

Comment:
Please just copy over the description rather than inherit. Makes it easy for a human to read the code and know what it does straight away.